frontend/pages/_app.tsx

```diff
@@ -1,27 +1,29 @@
-import Head from "next/head";
-import React, {useEffect} from 'react';
-import "../../modularhistory/static/styles/base.scss";
+import { Provider } from 'next-auth/client';
+import { AppProps } from 'next/app';
+import Head from 'next/head';
+import React, { useEffect } from 'react';
+import '../../modularhistory/static/styles/base.scss';
 
-function MyApp({ Component, pageProps }) {
+const App = ({ Component, pageProps }: AppProps) => {
   useEffect(() => {
     // Remove the server-side injected CSS.
-    // See https://github.com/mui-org/material-ui/blob/master/examples/nextjs/
+    // See https://github.com/mui-org/material-ui/blob/master/examples/nextjs/.
     const jssStyles = document.querySelector('#jss-server-side');
     if (jssStyles) {
       jssStyles.parentElement.removeChild(jssStyles);
     }
+    // TODO: get Django CSRF cookie?
+    // const url = "/api/csrf/set";
+    // axios.get(url).then(console.log);
```

Comment:
Does this need a new issue in Jira / is the authentication functionally complete without it?

Reply:
Ah. I'll create a Jira issue to circle back to it. Auth functionality is complete without it, BUT... Django only sends the CSRF cookie token when you visit a Django page. If we Reactify everything, it's possible that someone could visit the site and not receive a CSRF cookie from Django before it's required to make some POST request with DRF.